src/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -70,11 +70,11 @@ internal static Type GetTypeHelper(Type typeStart, Type[]? genericArgs, IntPtr p
 
         public static bool operator ==(RuntimeTypeHandle left, object right) { return left.Equals(right); }
 
-        public static bool operator ==(object left, RuntimeTypeHandle right) { return right.Equals(left); }
+        public static bool operator ==(object? left, RuntimeTypeHandle right) { return right.Equals(left); }
 
         public static bool operator !=(RuntimeTypeHandle left, object right) { return !left.Equals(right); }
 
-        public static bool operator !=(object left, RuntimeTypeHandle right) { return !right.Equals(left); }
+        public static bool operator !=(object? left, RuntimeTypeHandle right) { return !right.Equals(left); }
```

Comment:
You also need question marks on `object? right` in the first == above and the first != above.

Reply:
Argh. Good catch 😄